### DISCUSSION TITLE
Language

### DISCUSSION THREAD

**Discussion by iezed** [maritaca-ai/maritalk-api#87] [General]
Hello, do you have any plans to add English language support? Is the service currently active? What would the prices be in Euros or Dollars? Thank you.

**Comment by rodrigo-f-nogueira** [comment]
Hi @iezed,

Thanks for your interest. Yes, the service is active and we plan to release an English documentation soon.

The price in USD is 1.80 USD per one million tokens (input or output) for our flagship model, Sabiá-3.

**Comment by peq42** [comment]
when will there be an english UI for the chat page? @rodrigo-f-nogueira 
I can't really recommend it for most people outside Brazil if its only in portuguese

**Comment by peq42** [comment]
up?

**Comment by peq42** [comment]
@rodrigo-f-nogueira

**Comment by rodrigo-f-nogueira** [comment]
Hi @peq42 we are working on it. Thanks for the reminder.
